### PR TITLE
Replace bullets with numbers

### PR DESCRIPTION
**Pull request by rachelmyers**
We're having trouble sticking to the roadmap. The hope behind this PR is that if the roadmap was easier to reference, as in "We're working toward section 0.1.3 right now; please hold off on other PRs" we'll stick to the roadmap.

Signed-off-by: Rachel Myers <rachelmyers@google.com>

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -4,45 +4,45 @@ The CloudEvents Roadmap.
...
-* Document at least 3 sample events that conform to the specification.
-* Github repo is organized to be approachable to a engineers who might want to
+1. Draft specification that WG members agree *could* provide interoperability.
+1. Document at least 3 sample events that conform to the specification.
+1. Github repo is organized to be approachable to a engineers who might want to
 implement the spec.
-* Additional materials:
-  * Use-cases.
-  * Placeholder for protocol specification, folder and brief starter doc for
+1. Additional materials:
```
[the remark this answers is not in the document]
Comment (ultrasaurus) [reply roadmap.md]: Yeah, I saw that too.  If markdown is not expressive enough. I would be ok with dropping "Additional Materials" header.

File: roadmap.md [hunk 2]
```diff
@@ -4,45 +4,47 @@ The CloudEvents Roadmap.
...
 *0.3*
 
-* Interoperability demo.
-  * At least one open source implementation of sending and receiving events.
-  * Events are sent by code written by Developer1 and recieved by code written by Developer2, where Developer1 has no knowledge of Developer2.
-  * Draft trigger specification (e.g. how events + actions are associated).
-* Draft documentation and user guides.
+1. Interoperability demo.
+
+  i. At least one open source implementation of sending and receiving events.
```
Comment (duglin) [inline roadmap.md]: These are kind of funky. Look at the rendered version, these appear to the left of their parent numbering.

**Comment by duglin** [comment]
@rachelmyers can you fix the DCO issue?

File: roadmap.md (hunk 2 above)
Comment (duglin) [reply roadmap.md]: And the numbering is off a little too.

File: roadmap.md [hunk 3]
```diff
@@ -2,47 +2,48 @@
...
 
-* Interoperability demo.
-  * At least one open source implementation of sending and receiving events.
-  * Events are sent by code written by Developer1 and recieved by code written by Developer2, where Developer1 has no knowledge of Developer2.
-  * Draft trigger specification (e.g. how events + actions are associated).
-* Draft documentation and user guides.
+1. Interoperability demo.
+    1. At least one open source implementation of sending and receiving events.
+    1. Events are sent by code written by Developer1 and recieved by code written by Developer2, where Developer1 has no knowledge of Developer2.
+    1. Draft trigger specification (e.g. how events + actions are associated).
```
Comment (rachelmyers) [inline roadmap.md]: We agreed to alphabetical lists, but it's not supported by [GitHub Flavored Markdown](https://github.github.com/gfm/#example-265). This indentation fixes the rendering. [Rendered view](https://github.com/rachelmyers/spec/blob/75e6a3fd4fdf23e5bab09364bdc685a9869bfcd3/roadmap.md).

**Comment by markpeek** [comment]
LGTM

**Comment by duglin** [comment]
LGTM
Approved to these changes during the 4/12 call.

**Pull request merged**